src/ml/surrogate.hpp

```diff
@@ -147,7 +147,7 @@ PERFFASPECT()
   inline void _evaluate(long num_elements,
                         long num_in,
                         size_t num_out,
-                        const TypeInValue** inputs,
+                        TypeInValue** inputs,
```

Comment:
Why is const dropped?

Reply:
I forced pushed another version and made it more coherent, without requiring changing the library header file. Resource Manager that calls umpire::deallocate seems to fail when called with constant pointers.